torch/csrc/jit/passes/onnx/preclude_inlining.cpp

```diff
@@ -0,0 +1,65 @@
+#include <torch/csrc/jit/passes/onnx/preclude_inlining.h>
+#include <torch/csrc/jit/jit_log.h>
+#include <torch/csrc/jit/passes/onnx/helper.h>
+
+namespace torch {
+namespace jit {
+
+void functionCallSubstitution(Block* block) {
+  for (auto it = block->nodes().begin(), end = block->nodes().end();
+       it != end;) {
+    Node* cur = *it++;
+    if (cur->kind() == prim::CallFunction) {
```

Comment:
sorry for different commentary for different reviewers, but switching on the kind (as you were doing earlier) is more idiomatic. 
```
switch (cur->kind()):
     case prim::CallFunction
```